web-wallet/src/routes/(app)/dashboard/+layout.svelte

```diff
@@ -56,6 +56,12 @@
     networkStatusIconPath = mdiLink;
     syncStatusLabel = `Dusk ${networkName}`;
   }
+
+  let networkName = "Unknown";
```

Comment:
For clarity, this is better moved before the reactive statements as the variable is used there.